javascript/javascript3/week2/javascript3-homwork-week2/Giphy api/app.js

```diff
@@ -0,0 +1,26 @@
+const API_KEY = "OaG19WwiSmErWJe7q1FagPpJ3Re3otCH";
```

Comment:
No API Key in plain text

Reply:
I changed the key in the last classroom😒